### ISSUE TITLE
Client Runs page suggestions missing cookbooks

### ISSUE THREAD

**Issue by lancewf** [chef/automate#690]
# User Story

As an Automate User, I want all the cookbooks in all the nodes run lists to appear as suggestions for cookbooks in the Client Runs page's search bar. 

_The below image comes from a bank. It is showing the node details run list of a node that has the cookbook "db_chef_server_ha"._
<img width="322" alt="Screen Shot 2019-06-26 at 10 35 44 AM" src="https://user-images.githubusercontent.com/1679247/60205725-cee74700-9806-11e9-962c-dcd3ab3c64fa.png">

The problem is shown when the user goes to the Client Runs search bar and the "db_chef_server_ha" cookbook is not listed in the suggestions. 

The search bar's suggestions are filled from a CCR's "node.automatic.cookbooks" (https://github.com/chef/automate/blob/master/components/ingest-service/examples/converge-success-report.json#L2727).

The run list is filled from the "run_list" field on a CCR https://github.com/chef/automate/blob/master/components/ingest-service/examples/converge-success-report.json#L3035. 

We need to first ensure the CCR "node.automatic.cookbooks" field always contains cookbooks from the "run_list" field. If this is not the case then during ingestion we need to add the cookbooks from the run list to the list of cookbooks stored for each node. 

### References
Below is a link to the original issue. 
https://chefio.atlassian.net/browse/A2-893

# Definition of Done

Cookbooks from a nodes' run list are always suggestable in the Client Runs search bar. 

# Demo Script / Repro Steps

We do not know the exact problem, so we can not have reproducible steps. 
If a CCR's "node.automatic.cookbooks" does not always have all the cookbooks from the run list then the below steps will repro the problem. 
1. `start_all_services`
1. `send_chef_run_example`
1. Open https://a2-dev.test/client-runs
1. Search for Cookbook "insights-test"

**Comment by lancewf** [comment]
I am having problems replicating this bug with cookbooks. I am seeing a similar problem with attributes. I am going to investigate the problem of the attributes because the bug might only occur when there are hundreds of cookbooks which I have not tested.

**Comment by lancewf** [comment]
I have been able to replicate the bug with cookbooks by having over 105 cookbooks in a run_list. I also found where the problem is, this link https://github.com/chef/automate/blob/master/components/config-mgmt-service/backend/elastic/suggestions.go#L102 the ``SuggestionQuerySize`` is set to 100. If you increase this value to 200 cookbooks over 105 will be added to suggestions. 

My first guess on how to fix this problem is to use the text typed in to limit the aggregation.
